plugins/node.d/icinga

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf
+
+ICINGACLI=${ICINGACLI:-$(which icingacli)}
+JQ=${JQ:-$(which jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ "$ICINGACLI" != ""  -a "$JQ" != "" ] ; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli or jq)"
+        exit 1
+    fi
+fi
+export TERM=vt100
+
+if [ "$1" = "config" ]; then
+
+    echo "multigraph icinga_hosts"
```

Comment:
I think `icinga_host_checks` would be a bit more specific. Maybe in the future the plugin will also support "icinga hosts" in a cluster (or something else host-related) ...
(but this is a matter of taste)

Reply:
is renamed